index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>My Portfolio</title>
+</head>
+<body>
+  <header class="header">
```

Comment:
- [ ]  As the previous reviewer has earlier suggested,  make sure the logo and the navbar are vertically aligned (`align-items: center`) and also use `justify-content: space-between` to ensure that the elements are positioned extremely opposite each other at the side of the header section. 

Currently, your hamburger is not at the extreme right side of the header section 


| Figma Design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/212179311-503b65a2-e2a7-47f6-99b6-b8a8a48f6cbd.png)  | ![image](https://user-images.githubusercontent.com/63473584/212179082-522020ee-b31c-4ba5-a996-6624e932eecc.png)  |